packages/fxa-content-server/app/scripts/views/pair/index.js

```diff
@@ -21,22 +22,27 @@ class PairIndexView extends FormView {
     const uap = this.getUserAgent();
     const isFirefoxDesktop = uap.isFirefoxDesktop();
 
-    if (! isFirefoxDesktop) {
-      // other browsers show an unsupported screen
+    if (! isFirefoxDesktop || ! this.broker.hasCapability('supportsPairing')) {
+      // other browsers show an unsupported screen or if no capability to pair
       return this.replaceCurrentPage('pair/unsupported');
     }
 
+    const account = this.broker.get('browserSignedInAccount');
     // If we reach this point that means we are in Firefox Desktop
-    if (! this.broker.get('browserSignedInAccount')) {
-      // if we are not logged into Sync then we offer to sign in
+    if (! account) {
+      // if we are not logged into Sync then we offer to connect
       return this.replaceCurrentPage('connect_another_device');
```

Comment:
Is connect another device correct here? Wouldn't we want the user to sign in?

Reply:
ah yes! We can now use the new `navigateAway` thing here as well, good catch!